packages/codebytes/src/index.tsx

```diff
@@ -1,24 +1,79 @@
+import { Box, IconButton } from '@codecademy/gamut';
+import { FaviconIcon } from '@codecademy/gamut-icons';
+import { Background, states, system } from '@codecademy/gamut-styles';
+import { StyleProps } from '@codecademy/variance';
+import styled from '@emotion/styled';
 import React, { useState } from 'react';
 
+import { languageOption } from './consts';
 import { Editor } from './editor';
 
+const editorStates = states({
+  isIFrame: { height: '100vh' },
+});
```

Comment:
The states utility allows us to dynamically set styles based on props. This is slightly different from our conventional way of passing in props, since we are using system.css. Main takeaway is that the EditorContainer can work with editorBaseStyles, but will apply editorStates if a prop value is true. 

https://gamut.codecademy.com/?path=/docs/foundations-system-variants--page#states

Reply:
In what scenario would `isIFrame` be set to true and this styling apply?

Reply:
@zippyzow It will be set to true when render it from Next.js. When rendered on the forums, the parent iFrame will need Codebytes it to take up the entire viewport height. This is the current implementation.

Reply:
Thanks for the explanation!